openmdao/utils/file_utils.py

```diff
@@ -439,6 +444,10 @@ def _get_outputs_dir(obj=None, *subdirs, mkdir=True):
     The resulting outputs directory will be nested where each problem's output directory
     contains its own output files and subdirectories as well as any subproblems.
 
+    This directory also will include a .openmdao_outputs hidden file that
```

Comment:
This is a useful identifier, though I wonder if we should warn people not to use that filename for anything else.

Reply:
We should, but I'm going to write more thorough documentation of the output directory as part of part 3 of the POEM, since POEM 097 will be fully implemented at that point.